### DISCUSSION TITLE
Welcome to youtube-dl-wpf Discussions!

### DISCUSSION THREAD

**Discussion by database64128** [database64128/youtube-dl-wpf#54] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kimycai** [comment]
hello nice developer,may i say something here?
I want to talk about the custom download path.
When I put the new path with the new folder,the download button will not be useable,so could the download button be useable when i put the  new path with the new folder and create the new folder?

**Reply by database64128** [reply]
Hi, you're supposed to use the Browse button to navigate and create new folders if necessary. It was an intentional design choice to make the download button gray out when the specified download path doesn't exist, so you don't end up accidentally downloading into a new folder when there's a typo in the download path.

**Comment by kimycai** [comment]
hi nice developer，i want to talk about the "custom download path"again.Could it also be a droplist button-like that can remember the historical path so that I can select the path I put before easily without typing the  path .

**Reply by database64128** [reply]
This does sound like something nice to have. I'll see what I can do. Can you open an issue so I won't forget when I have time to work on this project?

**Reply by kimycai** [reply]
let me try.

**Reply by database64128** [reply]
#93
